README.md

```diff
@@ -119,6 +119,9 @@ await ReportUtility.addExpectedResult("Invalid username or password message is d
 	AssertionUtility.expectEqual(await loginPage.getMessagePopup().getTextMessage(), "Invalid username or password");
 });
 ```
+#### Traceability
+
+See [traceability](TRACEABILITY.md) page for details on how to add into Allure Reporting traceability of specs against test cases.
```

Comment:
The usual collocations for "trace" appear to be "trace between A and B" and "trace A with B", thus I suggest "traceability between specs and test cases" of "traceability of specs with test cases", as you prefer

Reply:
Applied rewording